src/vaults/vaults.controller.ts

```diff
@@ -24,6 +28,29 @@ export class VaultsController {
     return this.vaultService.listVaults(Chain.getChain(chain), currency);
   }
 
+  @Get('/harvests')
```

Comment:
these are pretty heavy methods we will probably want to use `UseCache()`

Reply:
u right, we should do some for optimization